### PR TITLE
Added a suggestion in Setup Linking

### PR DESCRIPTION
**Pull request by frontegi**
Just spent 2 hours on checking every configuration (AWS, Alexa Dev, Home Assistant logs...) , then figured out that active IP BAN was preventing to continue with the linking.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @frontegi, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22660).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/alexa.smart_home.markdown [hunk 1]
```diff
@@ -217,6 +217,7 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
   - Click the Smart Home skill you just created.
   - Click `Enable to use`.
   - A new window will open to direct you to your Home Assistant's login screen.
+    note: If you get a 403 Unauthorized at this stage, please verify if you have IP Ban activated.
```
Comment (frenck) [inline source/_integrations/alexa.smart_home.markdown]: I'm not sure if this comment is clear or what the intention is?

Right now it reads like: You need to activate it?

Additionally, it doesn't matter if IP Ban is activated or not?

Comment (frontegi) [reply source/_integrations/alexa.smart_home.markdown]: Hi @frenck , you are right, statement not completely clear. The problem is that after some tests, activated IP ban could prevent the conclusion of the procedure. I will try to better explain it.

**Review by frontegi — COMMENTED** [review]
Now the statement seems to be more clear.

File: source/_integrations/alexa.smart_home.markdown [hunk 2]
```diff
@@ -217,7 +217,9 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
   - Click the Smart Home skill you just created.
   - Click `Enable to use`.
   - A new window will open to direct you to your Home Assistant's login screen.
-    note: If you get a 403 Unauthorized at this stage, please verify if you have IP Ban activated.
+    
+    note: If you have [IP Ban](https://www.home-assistant.io/integrations/http/#ip_ban_enabled) activated, after many try, there is the possibility receive a "403 Forbidden" return code from the Home assistant's login screen.
```
Comment (frenck) [inline source/_integrations/alexa.smart_home.markdown:221]: So, how would many tries happen?

Some additional notes:

- Start sentences with capital letters.
- Don't use full URLs, instead, make URLs from the website root.

File: source/_integrations/alexa.smart_home.markdown [hunk 3]
```diff
@@ -217,7 +217,9 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
...
+    In this case, temporarily disable [IP Ban](https://www.home-assistant.io/integrations/http/#ip_ban_enabled) to conclude the procedure or remove the banned IP entry from the list.
```
Comment (frenck) [inline source/_integrations/alexa.smart_home.markdown]: Let's not recommend disabling it.

Comment (frontegi) [reply source/_integrations/alexa.smart_home.markdown]: Right. sorry, learning how to proactively contribute ! :-)

**Review by frontegi — COMMENTED** [review]
As suggested, removing a suggestions that can potentially expose the HA installation to attackers.
Added link to the blocked list feature documentation.

**Pull request closed without merging**